tools/rtrclient.1

```diff
@@ -27,7 +27,7 @@ rtrclient \- rtr rpki client
 .IR PRIVATE_KEY
 [\fIHOST_KEY\fR]
 .SH DESCRIPTION
-\fBrtrclient\fR allows to connect to an rtr server and print prefix, origin AS and router key updates.
+\fBrtrclient\fR connects to an rtr server and print prefix, origin AS and router key updates.
```

Comment:
is *rtr server* the correct term here? Maybe write:

```
connects to a RPKI cache server with RTR protocol and prints prefix, origin AS and router key updates. 
```

Reply:
phew, now it gets complicated. Final version: "connects to an RPKI/RTR cache server and prints prefix, origin AS, and router key updates. "

Reply:
updated, including the oxford comma :)